### PR TITLE
add new note for ephemeral storage calculations

### PR DESCRIPTION
**Pull request by rphillips**
https://github.com/kubernetes/kubernetes/issues/111965 brings up a concern about ephemeral storage calculations. I took the concern to Sig-Node and the collective thought is that mounting disks to /var/lib/kubelet and /var/lib/containers is not necessarily supported, and definitely not supported with ephemeral storage calculations.

This PR adds a blurb that ephemeral storage is not reported correctly in this instance.

### CODE REVIEW

File: content/en/docs/concepts/configuration/manage-resources-containers.md
Comment (mrunalp) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:328]: nit: Should we drop `by design`?

Comment (rphillips) [reply content/en/docs/concepts/configuration/manage-resources-containers.md]: updated

File: content/en/docs/concepts/configuration/manage-resources-containers.md
Comment (mrunalp) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:328]: ...mount /var/lib/containers or /var/lib/kubelet on separate disks..

Comment (rphillips) [reply content/en/docs/concepts/configuration/manage-resources-containers.md]: updated.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-node-pr-reviews : Please could you provide signoff from a technical standpoint if the content is what you've agreed upon with the assignee? This looks okay to me from a docs perspective & have therefore added the lgtm label.

**Comment by jimangel** [comment]
Thanks for the PR @rphillips! I agree that this topic could be clarified further, but I feel like this subject is addressed in the following sentence (under single file system: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#configurations-for-local-ephemeral-storage):

> Typically, both `/var/lib/kubelet` and `/var/log` are on the system root filesystem, and the kubelet is designed with that layout in mind.

Could we potentially clarify that line?

In addition, the current PR opens a "{{ note }}" shortcode below another note. Pictured:

---

image

---

If you reject my suggestion above, please consider combining notes or changing the location. My feedback is non-blocking if another approver wants to merge this as-is.

/lgtm

**Review by mrunalp — APPROVED** [review]

**Comment by reylejano** [comment]
Thank you for addressing feedback
/lgtm
/approve

**Pull request merged**
